examples/testing/main_test.go

```diff
@@ -66,3 +66,12 @@ func TestIntMinTableDriven(t *testing.T) {
 		})
 	}
 }
+
+// Benchmark tests typically go in `_test.go` files and begin with `Benchmark`.
+// Each benchmark function is run several times, with `b.N` increasing on each
+// run until the `testing` runner deems the benchmark to be accurate.
+func BenchmarkIntMin(b *testing.B) {
```

Comment:
Hmm, I would hope for a clearer explanation of `b.N`. What the reader needs to know is that they have to run their "thing" in a loop `b.N` times

Reply:
Hopefully addressed in bf78b89.